libvast_test/vast/test/fixtures/events.hpp

```diff
@@ -63,7 +65,13 @@ struct events {
   }
 
   std::vector<table_slice_ptr> copy(std::vector<table_slice_ptr> xs);
+
+  auto take_n(const std::vector<table_slice_ptr>& xs, size_t n) {
+    VAST_ASSERT(n <= xs.size());
+    auto first = xs.begin();
+    auto last = first + n;
+    return std::vector<table_slice_ptr>{first, last};
```

Comment:
Does anything speak against returning a `span<table_slice_ptr>` here instead?

Reply:
I would not expect a function named `take` to return a *view* into the data. Besides, for unit tests I think operating on owning containers makes life easier. There's no reason to optimize memory allocations here.